lib/her/model/attributes.rb

```diff
@@ -25,6 +25,7 @@ def initialize(attributes={})
 
         attributes = self.class.default_scope.apply_to(attributes)
         assign_attributes(attributes)
+        @changed_attributes = {} if persisted?
```

Comment:
This should be covered in [#404](https://github.com/remiprev/her/pull/411/files#diff-7280373369c4cbefb4c1e7c1df8b8a36R51).